deploy/charts/external-secrets/README.md

```diff
@@ -47,6 +47,7 @@ The command removes all the Kubernetes components associated with the chart and
 | certController.image.pullPolicy | string | `"IfNotPresent"` |  |
 | certController.image.repository | string | `"ghcr.io/external-secrets/external-secrets"` |  |
 | certController.image.tag | string | `""` |  |
+| certController.image.falvour | string | `""` | The falvour of the image tag to use. The default is distroless. |
```

Comment:
```suggestion
| certController.image.flavour| string | `""` | The flavour of the image tag to use. The default is distroless. |
```

Reply:
I think you can use `make helm.docs` to generate this update to the chart README- this way this is done programatically and we avoid typos.